### PR TITLE
[samples/HelloWorld] Add Hello World sample.

### PR DESCRIPTION
**Pull request by jonpryor**
Update `README.md` to describe how to build, install, and run the sample.

### CODE REVIEW

File: samples/HelloWorld/MainActivity.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+﻿using Android.App;
+using Android.Widget;
+using Android.OS;
+
+namespace HelloWorld
+{
+	[Activity (
+			Icon            = "@mipmap/icon",
```
Comment (dellis1972) [inline samples/HelloWorld/MainActivity.cs]: @jonpryor the indenting looks a bit weird

**Comment by jonpryor** [comment]
> the indenting looks a bit weird

Is two tab stops too many? Would 1 tab stop be better?

**Comment by dellis1972** [comment]
I think 1 looks better

**Comment by jonpryor** [comment]
@dellis1972 updated.

**Pull request merged**